test/sequential/test-repl-persistent-history.js

```diff
@@ -147,6 +150,22 @@ const tests = [{
   test: [UP, UP, UP, CLEAR],
   expected: [prompt, convertMsg, prompt, prompt + '\'=^.^=\'', prompt]
 },
+{
+  before: function() {
+    try {
+      fs.mkdirSync(historyPathFail);
+    } catch (err) {}
+  },
+  after: function() {
+    try {
+      fs.rmdirSync(historyPathFail);
+    } catch (err) {}
```

Comment:
If this fails the test should fail here, since it will cause a more ambiguous error later.